src/components/filters/FilterContainer.vue

```diff
@@ -13,7 +13,7 @@
   }
 
   .filter-header {
-    background-color: #e4e4e4;
+    background-color: #f5f5f5;
```

Comment:
magic number ? add comment what this hard coded color does

Reply:
It is now the same color gray as the biobank headers.
But added a comment mentioning this